src/_posts/databases/postgresql/2000-01-01-managing.md

```diff
@@ -262,8 +258,39 @@ Creating a new database user is only available from the database dashboard.
    User "my_user" created with password "YANs3y07m5_KJC2MSDGebh8tx1lliFWh2Yb239zVqGQvbElWDjIN7QWspVH92Ul8".
    ```
 
+### Update a User Password
+
+{% note %}
+Updating a User Password is only available from the command line
+{% endnote %}
 
-### Deleting an Existing User
+#### Using the Command Line
+
+1. Make sure you have correctly [setup the Scalingo command line tool]({% post_url platform/cli/2000-01-01-start %})
+2. From the command line, list the addons attached to your app:
+   ```bash
+   scalingo --app my-app addons
+   ```
+   The output should look like this:
+   ```text
+   +------------+-----------------------------------------+-------------------------+---------+
+   |   ADDON    |                   ID                    |          PLAN           | STATUS  |
+   +------------+-----------------------------------------+-------------------------+---------+
+   | PostgreSQL | ad-50d873b0-7760-4a4b-8d1a-760ec997b481 | postgresql-starter-1024 | running |
+   +------------+-----------------------------------------+-------------------------+---------+
+   ```
+3. Locate the `ID` of the addon for which you want to update a user password
+4. Update the user password with the following command:
+   ```bash
+   scalingo --app my-app --addon <addon_id> database-users-update-password <username>
+   ```  
+5. An interactive prompt asks you for the new password you want to attribute to your user
```

Comment:
I like the precise explanation from create user, what about using it here too?

```suggestion
5. An interactive prompt asks you for a password you want to attribute to your user:
   - Either chose a password and confirm it
   - Or let the platform generate a password by leaving the field empty

   The output should look like this:
   - If you set a password:
   ```text
   User "my_user" created.
   ```
   - If you let the platform generate a password:
   ```text
   User "my_user" created with password "YANs3y07m5_KJC2MSDGebh8tx1lliFWh2Yb239zVqGQvbElWDjIN7QWspVH92Ul8".
   ```

```

Reply:
I didn't know you implemented the password generation client sided! Adding this